content/articles/transferring-domain.markdown

```diff
@@ -96,3 +96,7 @@ We suggest pointing [your domain to our name servers before the transfer](/artic
 Most transferred domains will be extended the minimum required extension period. For example, `.com` domains will always be extended one year when they are transferred.
 
 The price of this extension is included in the transfer fee.
+
+### DS records
+
+Before delegating your domain to DNSimple make sure that you first [enable DNSSEC for the domain](/articles/dnssec/#enabling-dnssec) in order to safely remove the DS records from your current provider.
```

Comment:
I think we also need to make this suggestion optional only for those customers using DNSSEC in their old providers.